cognite_toolkit/_cdf_tk/storageio/_base.py

```diff
@@ -50,6 +59,11 @@ def data_to_json_chunk(self, data_chunk: T_WritableCogniteResourceList) -> list[
     def json_chunk_to_data(self, data_chunk: list[dict[str, JsonVal]]) -> T_CogniteResourceList:
         raise NotImplementedError()
 
+    @abstractmethod
+    def configurations(self, identifier: T_StorageID) -> Iterable[StorageIOConfig]:
+        """Return configurations for the storage item."""
+        raise NotImplementedError()
```

Comment:
This is a small adjustment that became apparent when writing the download command.